src/infrastructure/RequestHelper.ts

```diff
@@ -40,7 +40,9 @@ export async function get(
   // Rescurse through pagination results
   if (!query.page && underLimit && pagination.next) {
     const { next } = Li.parse(headers.link);
-    const more = await get(service, next.replace(`${service.url}/`, ''), {
+    const leaf = service.url.split('/').pop() || ""
```

Comment:
if the url was http://gitlab.com/api/v1, wouldnt leaf be v1?

Reply:
The regex handles it just fine:

See here: https://regex101.com/r/2LSP9L/1

Reply:
In that case leaf is projects, but what if leaf were v1, like the case I pointed about above.

Reply:
My dear sir, you can modify the regex :sweat:

Reply:
Lol I realised that after I commented.